### PR TITLE
Fix typo where 'sources' was 'source'

### PR DESCRIPTION
**Pull request by gnilekaw**
As the title suggests, this PR fixes a typo that was preventing pushes to Redshift. It also adds a convenience method, `StartSyncJob.force_sync`, which can be used to force a sync, which is helpful when you change the Redshift schema.

### CODE REVIEW

File: app/jobs/start_sync_job.rb [hunk 1]
```diff
@@ -1,12 +1,20 @@
 class StartSyncJob < ApplicationJob
   attr_accessor :sync, :part_size
 
-  def perform(sync_id, part_size = OrganizationExportJob::PART_SIZE)
+  PART_SIZE = OrganizationExportJob::PART_SIZE
+
+  def self.force_sync
+    sync = Sync.create state: SyncMicroMachine::UNSTARTED
```
Comment (jonallured) [inline app/jobs/start_sync_job.rb]: Oops, I forgot that this should actually start the sync in the `SyncMicroMachine::STARTING` state. My bad.

File: app/jobs/start_sync_job.rb [hunk 2]
```diff
@@ -1,12 +1,20 @@
 class StartSyncJob < ApplicationJob
   attr_accessor :sync, :part_size
 
-  def perform(sync_id, part_size = OrganizationExportJob::PART_SIZE)
+  PART_SIZE = OrganizationExportJob::PART_SIZE
```
Comment (jonallured) [inline app/jobs/start_sync_job.rb]: I can't think of a better way to do this, but it's def something I'm going to keep an eye out for.

Comment (gnilekaw) [inline app/jobs/start_sync_job.rb]: One option would be to define it an env var. Batches are configured for performance, right? So unless we're adjusting the batch size at runtime, why define it in the implementation?

Comment (jonallured) [inline app/jobs/start_sync_job.rb]: Whoa, interesting - have one batch size on Heroku and another locally, for instance. That sounds pretty dope!

Comment (gnilekaw) [inline app/jobs/start_sync_job.rb]: Should be a small change. I'll include it here.

File: app/jobs/organization_export_job.rb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 class OrganizationExportJob < ApplicationJob
-  PART_SIZE = 1_000
+  PART_SIZE = ENV['BATCH_EXPORT_SIZE']
```
Comment (jonallured) [inline app/jobs/organization_export_job.rb]: Are you cool with moving this into the `secrets.yml` file? I think we sorta decided that using ENV outside that file was going to be an anti-pattern on this app.

**Pull request merged**